controllers/commonservice_controller.go

```diff
@@ -72,9 +73,9 @@ func (r *CommonServiceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-		if isScoped := r.checkScope(csScope, req.NamespacedName.Namespace); !isScoped {
-			klog.Infof("CommonService CR %v is not in the scope", req.NamespacedName.String())
-			return ctrl.Result{}, nil
+		if isScoped = r.checkScope(csScope, req.NamespacedName.Namespace); !isScoped {
+			klog.Infof("CommonService CR %v is not in the scope, only reconciles its configuration of cluster scope resource", req.NamespacedName.String())
+			// return ctrl.Result{}, nil
```

Comment:
Is this comment intentional？

Reply:
Oh in the previous design, we will directly finish reconciliation if it is out of scope. Updated to remove it